osbs/cli/main.py

```diff
@@ -313,11 +315,20 @@ def _print_build_logs(args, osbs, build):
         print("Build submitted (%s), watching logs (feel free to interrupt)" % build_id)
         try:
             for line in build_logs:
+                if user_warnings.is_user_warning(line):
+                    user_warnings.store(line)
+                    continue
+
                 print('{!r}'.format(line))
         except Exception as ex:
             logger.error("Error during fetching logs for build %s: %s", build_id, repr(ex))
 
         osbs.wait_for_build_to_finish(build_id)
+
+        if str(user_warnings):
```

Comment:
:eyes:

Reply:
`__len__`, `__bool__` ?

Reply:
It allows us to control if there is some content in user_warnings. Is there something wrong?

Reply:
Good point :+1: